components/sync15/Cargo.toml

```diff
@@ -6,24 +6,48 @@ authors = ["application-services@mozilla.com"]
 license = "MPL-2.0"
 exclude = ["/android", "/ios"]
 
+[features]
+# The default feature is what mozilla-central wants to make vendoring/patching easier.
+default = ["sync-engine"]
+
+random-guid = ["sync-guid/random"]
+
+# Some consumers of this just need our encrypted payloads and no other sync functionality.
+crypto = ["rc_crypto", "base16", "base64"]
+
+# Some crates need to implement a "sync engine", but aren't a "sync client" (ie, their
```

Comment:
can we link to our docs that define what `engine` is here, https://mozilla.github.io/application-services/book/design/sync-manager.html?highlight=engine#clients-engine maybe?

That said, maybe it's better this goes in the `README.md` of this crate